docs/source/json-api/index.rst

```diff
@@ -1064,6 +1064,120 @@ HTTP Response
       "status": 200
     }
 
+List All DALF Packages
+**********************
+
+HTTP Request
+============
+
+- URL: ``/v1/packages``
+- Method: ``GET``
+- Content: <EMPTY>
+
+HTTP Response
+=============
+
+.. code-block:: json
+
+    {
+      "result": [
+        "c1f1f00558799eec139fb4f4c76f95fb52fa1837a5dd29600baa1c8ed1bdccfd",
+        "733e38d36a2759688a4b2c4cec69d48e7b55ecc8dedc8067b815926c917a182a",
+        "bfcd37bd6b84768e86e432f5f6c33e25d9e7724a9d42e33875ff74f6348e733f",
+        "40f452260bef3f29dede136108fc08a88d5a5250310281067087da6f0baddff7",
+        "8a7806365bbd98d88b4c13832ebfa305f6abaeaf32cfa2b7dd25c4fa489b79fb"
+      ],
+      "status": 200
+    }
+
+Where ``result`` is the JSON array containing package IDs of all loaded DALFs.
+
+Download a DALF Package
+***********************
+
+HTTP Request
+============
+
+- URL: ``/v1/packages/download``
+- Method: ``POST``
```

Comment:
How hard would this be?
```suggestion
- URL: ``/v1/packages/download/<package-id>``
- Method: ``GET``
```
That would greatly simplify downloading DALFs.

Reply:
this would be inconsistent with the query and fetch endpoints and would require to deal with akka DSL for routes, which I tried to avoid for as long as I could :)

Reply:
if we really want to have REST-like endpoints, we should change it to:
GET /v1/packages -- list packages
GET /v1/packages/<id> -- download package
POST /v1/packags -- upload

Reply:
@hurryabit I have implemented it as I specified in the previous comment.